### PR TITLE
PLT-4790: Mention don't appear linked in message box preview mode.

### PR DESCRIPTION
**Pull request by Hyeongmin-Kwon**
#### Summary
Solved mention like `@someone` didn't appear linked in message box preview mode.

#### Ticket Link
[GitHub](https://github.com/mattermost/mattermost-server/issues/6127)
[Jira](https://mattermost.atlassian.net/browse/PLT-4790)

### CODE REVIEW

File: components/post_view/post_message_view/post_message_view.jsx [hunk 1]
```diff
@@ -5,20 +5,15 @@ import PropTypes from 'prop-types';
...
 
-import {Parser, ProcessNodeDefinitions} from 'html-to-react';
+import {Posts} from 'mattermost-redux/constants';
+import {getChannelsNameMapInCurrentTeam} from 'mattermost-redux/selectors/entities/channels';
 
 import store from 'stores/redux_store.jsx';
 
 import * as PostUtils from 'utils/post_utils.jsx';
 import * as TextFormatting from 'utils/text_formatting.jsx';
 import * as Utils from 'utils/utils.jsx';
```
Comment (saturninoabril) [inline components/post_view/post_message_view/post_message_view.jsx]: This import arrangement made the `post_type.test.jsx` fail. Import should be arranged like:
```
import store from 'stores/redux_store.jsx';

import * as PostUtils from 'utils/post_utils.jsx';
import * as TextFormatting from 'utils/text_formatting.jsx';
import * as Utils from 'utils/utils.jsx';

import {Posts} from 'mattermost-redux/constants';   // eslint-disable-line import/order
import {getChannelsNameMapInCurrentTeam} from 'mattermost-redux/selectors/entities/channels';   // eslint-disable-line import/order
```

Comment (saturninoabril) [reply components/post_view/post_message_view/post_message_view.jsx]: Also, please do update snapshot of `tests/plugins/post_type.test.jsx` and see to it that it passed the tests.

**Comment by saturninoabril** [comment]
@Hyeongmin-Kwon Thanks for updating, I guess I'll just have to send separate patch for jest snapshot.

**Comment by grundleborg** [comment]
@Hyeongmin-Kwon Thanks for the PR. Please can you rebase this onto the latest `master`? Then we can go ahead and merge it.

**Pull request merged**